tests/integration_tests/integration_settings.py

```diff
@@ -39,6 +39,17 @@
 # to this format internally; in this case, to "None::ubuntu::focal::20.04".)
 OS_IMAGE = "focal"
 
+
+# Determines unique image type or flavor to exercise if the cloud supports
+# image-type lookup for daily_image and released_images.
+#
+# One of the following:
+# - generic
+# - minimal
+# - Pro
+# - Pro FIPS
+OS_IMAGE_TYPE = "generic"
```

Comment:
After spending some time on this proposal. I can reject my suggestion for optional `<offer_name>` suffix in `OS_IMAGE` for two reasons:
1. offer names are actually platform/cloud specific. A "minimal" image offer in EC2 will not have the same offer/product name in Azure  or GCP. The [pycloudlib.cloud.ImageType](https://github.com/canonical/pycloudlib/blob/1!10.0.0/pycloudlib/cloud.py#L28) interprets image types per cloud to provide the right mapping of ImageType to cloud offering/product/image stream so that the integration test runners don't have to have detailed knowledge of a certain flavor/offer name or image per-platform.
2. providing an optional `<offer_name>` suffix in OS_IMAGE also requires making `<image_id>::` a leading optional value of `None` or `null`  in order to provide a trailing `<offer_name>`. This leads to complexity of image specification where it can take either 4 or 5 parts, and significant documentation around when to use the optional leading `None::` 

For these reasons, I think it makes sense to stick with a new `OS_IMAGE_TYPE` config value (or we could call it IMAGE_FLAVOR) to better represent this unique image stream.